modules/features/filters/src/main/java/au/com/shiftyjelly/pocketcasts/filters/FiltersTooltip.kt

```diff
@@ -0,0 +1,136 @@
+import androidx.compose.ui.graphics.Outline
+import androidx.compose.ui.graphics.Path
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.invisibleToUser
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Density
+import androidx.compose.ui.unit.LayoutDirection
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.compose.AppTheme
+import au.com.shiftyjelly.pocketcasts.compose.components.TextH30
+import au.com.shiftyjelly.pocketcasts.compose.components.TextP40
+import au.com.shiftyjelly.pocketcasts.compose.theme
+import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+import kotlin.math.sqrt
+import au.com.shiftyjelly.pocketcasts.images.R as IR
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+@Composable
+internal fun FiltersTooltip(
+    onClickClose: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier,
+        verticalArrangement = Arrangement.spacedBy(0.dp),
+    ) {
+        Box(
+            modifier = Modifier
+                .offset(y = 6.dp, x = -16.dp)
+                .align(Alignment.End)
+                .size(16.dp)
+                .background(
+                    color = MaterialTheme.theme.colors.primaryUi01,
+                    shape = UpwardTriangleShape,
+                )
+                .semantics {
+                    invisibleToUser()
+                },
+        )
+
+        Box(
+            modifier = Modifier
+                .background(
+                    color = MaterialTheme.theme.colors.primaryUi01,
+                    shape = RoundedCornerShape(4.dp),
+                )
+                .semantics(mergeDescendants = true) {},
+        ) {
+            Column(
+                modifier = Modifier.padding(24.dp),
+            ) {
+                TextH30(
+                    text = stringResource(LR.string.filters_tooltip_title),
+                    disableAutoScale = true,
+                )
+                Spacer(
+                    modifier = Modifier.height(8.dp),
+                )
+                TextP40(
+                    text = stringResource(LR.string.filters_tooltip_subtitle),
+                    color = MaterialTheme.theme.colors.primaryText02,
+                    disableAutoScale = true,
+                )
+            }
+            IconButton(
+                onClick = onClickClose,
+                modifier = Modifier.align(Alignment.TopEnd),
+            ) {
+                Icon(
+                    painter = painterResource(IR.drawable.ic_close),
+                    contentDescription = stringResource(LR.string.close),
+                    tint = MaterialTheme.theme.colors.primaryText01,
+                )
+            }
+        }
+    }
+}
+
+private object UpwardTriangleShape : Shape {
```

Comment:
The same logic here is already in the code https://github.com/Automattic/pocket-casts-android/blob/a78a52816d157ddf9f0ecfcc947a628c10b28021/modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/PodcastHeaderTooltip.kt#L90 Would it be better to create a reusable tooltip component rather than repeating the code?

Reply:
This shape i a triangle pointing down and I need to draw one pointing up

Reply:
Ashita and I were talking about this, and we now have a tooltip component. I've updated this PR to include it